components/list/OrganizationsList.vue

```diff
@@ -181,11 +185,36 @@ function buildQuery() {
   if (limit) { query.limit = limit; }
   if (skip) { query.skip = skip; }
 
-  // todo sort
+  if (ctx.sortBy) { query.sort = sort(ctx); }
 
   return query;
 }
 
+function sort(ctx) {
+  const sortFields     = [];
+  const  sortDirection =  ctx.sortDesc ? -1 : 1;
+
+  let sortField;
+
+  switch (ctx.sortBy) {
+    case 'government':
+      sortField = 'governmentName';
+      break;
+    case 'country':
+      sortField = 'countryName';
+      break;
+    case 'organizationTypeId':
+      sortField = 'typeName';
+      break;
+    default:
+      sortField = ctx.sortBy;
+  }
```

Comment:
same as for contacts dont use switches
```suggestion
  let sortField = ctx.sortBy

  if(sortField == 'government')         sortField = 'governmentName';
  if(sortField == 'country')            sortField = 'countryName';
  if(sortField == 'organizationTypeId') sortField = 'typeName';
 }
```